internal/start/start.go

```diff
@@ -1125,6 +1125,10 @@ EOF
 
 	// Start Studio.
 	if utils.Config.Studio.Enabled && !isContainerExcluded(utils.Config.Studio.Image, excluded) {
+		cwd, err := os.Getwd()
+		if err != nil {
+			return errors.Errorf("failed to get working directory: %w", err)
+		}
```

Comment:
This can be refactored and reused with https://github.com/supabase/cli/blob/develop/internal/start/start.go#L275